Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3-buster as base
+
+EXPOSE 5000
+ENTRYPOINT root/.local/bin/poetry run flask run --host=0.0.0.0
```

Comment:
Different opinions here and it doesn't make any logical difference, but I'm a fan of entrypoints at the bottom of the Docker stage since that's where they'll conceptually run (as in, after the RUN commands etc.), and I'd probably go even further and explicitly duplicate this into both the dev & prod stages since I don't necessarily expect those to remain the same forever (for example, if we introduced a proper production web server that would probably only be for the prod stage)